carservice/src/main/java/com/senlainc/warsaw/tyurin/util/builder/CraftsmanBuilder.java

```diff
@@ -0,0 +1,30 @@
+package com.senlainc.warsaw.tyurin.util.builder;
+
+import com.senlainc.warsaw.tyurin.dto.CraftsmanDto;
+import com.senlainc.warsaw.tyurin.entity.Craftsman;
+import org.springframework.stereotype.Component;
+
+import javax.validation.Valid;
+
+@Component
+public class CraftsmanBuilder {
```

Comment:
Usually this type of classes called mappers, not builders. Because it can be confusing since we have builder pattern and it is slightly different